client-projects/the-lumery/index.ts

```diff
@@ -5,24 +5,36 @@ import {
   TICKER_TAPE_SELECTOR,
   YT_DEFER_SELECTOR,
   WHO_WE_WORK_WITH_SECTION_SELECTOR,
+  CHANGING_TAGLINES_SELECTOR,
 } from "./selectors";
 import {
   onlyPlayWhenVisible,
   pauseAnimations,
   resumeAnimations,
+  prefersReducedMotion
 } from "./utils";
 import { ytdefer_setup } from "./ytdefer";
 import { createClientFilterButtons } from "./whoWeWorkWithFilter";
 import { setUpDropdowns } from "./navDropdown";
+import { swapOutTaglines } from "./swapOutTaglines";
 
 document.addEventListener("DOMContentLoaded", () => {
-  // nav dropdowns have a staggered animation effect on their children
-  setUpDropdowns();
-
-  // all pages have a hero heading animation.
-  // the hero's intersectionObserver setup is contained within the function.
-  heroHeadingAnimation();
+  if (!prefersReducedMotion()) {
+    // nav dropdowns have a staggered animation effect on their children
+    setUpDropdowns();
```

Comment:
This implies that `setUpDropdowns()` is only to do with _animating_ the dropdowns. Is that right? If so, I wonder if it should be renamed to something like `setUpDropdownAnimations()` or something?

Reply:
Yep, it should. Updated.